packages/usbutils.rb

```diff
@@ -23,7 +23,11 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+  end
+
+  def self.postinstall
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/hwdata/"
```

Comment:
`FileUtils.mkdir_p "#{CREW_PREFIX}/share/hwdata/"`?  Since this is a post install operation, no need to add to the install temp path.